code/modules/client/verbs/suicide.dm

```diff
@@ -21,6 +21,8 @@
 		if(mmi.brainmob)
 			mmi.brainmob.suiciding = suicide_state
 
+//SKYRAT EDIT REMOVAL BEGIN - SUICIDE_VERB
+/*
 /mob/living/carbon/human/verb/suicide()
```

Comment:
Can we perhaps have some feedback to the user about it being disabled followed by a message to the admins?

Reply:
I don't see any reason to act on a message of 'x tried to access the suicide verb but couldn't lol' so please no extra spam

Same with the primary armament token redeeming message on improper alrt, haven't seen anybody act on it yet, because why would we -- they didn't do anything wrong.

Reply:
NotRanged has a point here, I'd tend to agree with it.
Verdict, @Gandalf2k15 ? Will you want this change done?

Reply:
It's fine as is.